src/test.jest.testRunner.ts

```diff
@@ -164,8 +166,26 @@ async function askRunner(
     title: 'compiles',
   });
 
+  // Adding Jest functions to the environment, so that we can run Jest expect():toBe() in .ask files.
+  // See a sample in src/askscript/__tests__/00-documentation-examples/documentation01-complete_example.test.ask
+  const testResources = {
+    expect: resource({
+      type: any,
+      async resolver(actual: any): Promise<e.Matchers<any>> {
+        return e(actual);
```

Comment:
Please add a comment explaining that ideally we would reuse `test` and `expect` from the Jest environment instead of a separate package

Reply:
Done.